### PR TITLE
Add usage instructions on README

### PR DESCRIPTION
**Pull request by stavros11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
 # qibolab_platforms_qrc
+
```
Comment (andrea-pasquale) [inline README.md:2]:
```suggestion
# QRC platforms
```
A better title (?)

**Review by andrea-pasquale — APPROVED** [review]
Thanks @stavros11.
I just put a suggestion for the title of the README.

**Review by Jacfomg — APPROVED** [review]

**Comment by rodolfocarobene** [comment]
It's good for me. Only one thing: probably is implicit, but I would explicitly write that after the export one has to open a new terminal or source the .bashrc

**Comment by stavros11** [comment]
> It's good for me. Only one thing: probably is implicit, but I would explicitly write that after the export one has to open a new terminal or source the .bashrc

`export` takes effect (only) in the current terminal, so you don't need to open a new one. 
If you put it in `.bashrc` then indeed it will take effect from the next terminal unless you source it or export in the current.

**Review by rodolfocarobene — APPROVED** [review]

**Pull request merged**